Shoko.Server/API/v3/Controllers/DashboardController.cs

```diff
@@ -105,16 +105,17 @@ private static bool MissingBothTvDBAndMovieDBLink(SVR_AnimeSeries ser)
         [HttpGet("TopTags")]
         public List<Tag> GetTopTags()
         {
-            return GetTopTags(10);
+            return GetTopTags();
         }
 
         /// <summary>
-        /// Gets the top <para>number</para> most common tags visible to the current user 
+        /// Gets the top <para>number</para> of the most common tags visible to the current user.
         /// </summary>
-        /// <param name="number"></param>
+        /// <param name="number">The max number of results to return</param>
+        /// <param name="filter">The <see cref="TagFilter.Filter" /> to use. (Defaults to <see cref="TagFilter.Filter.AnidbInternal" /> | <see cref="TagFilter.Filter.Misc" /> | <see cref="TagFilter.Filter.Source" />)</param>
         /// <returns></returns>
         [HttpGet("TopTags/{number}")]
```

Comment:
I would need to look it up, but I believe you can make it {number?} to make it optional, which can let you remove the above method entirely.

Reply:
Yeah, apparently that works.